### DISCUSSION TITLE
Vaultwarden syncing error

### DISCUSSION THREAD

**Discussion by Mcb2023** [dani-garcia/vaultwarden#4167] [Q&A] [answered]
A few days ago our users started getting errors when syncing their VaultWarden passwords. Running on ECS/Fargate service.

Also getting the same socket errors as in this discussion- https://github.com/dani-garcia/vaultwarden/discussions/4090  

Any help would be appreciated. Thanks

**Comment by BlackDex** [comment, marked as answer]
Those logs tell me nothing. These do not seem to come from Vaultwarden.
The socket errors are already solved in the current `testing` tagged version.

The only item i can think of in regards to syncing is the release of the new Bitwarden clients, which seem to need a full logout and login again to solve the syncing issue you are probably revering too.

**Reply by Mcb2023** [reply]
Thank You

**Reply by chelming** [reply]
This solved my syncing issue on Chrome. Thanks!
